tests/login.spec.ts

```diff
@@ -0,0 +1,223 @@
+import { test, expect } from '@playwright/test';
+import { MainPage } from '../pages/main.page.ts';
+import { LoginPage } from '../pages/login.page.ts';
+import { ProfilePage } from '../pages/profile.page.ts';
+
+test.describe("Authorization form", () => {
+    let mainPage: MainPage;
+    let loginPage: LoginPage;
+    let profilePage: ProfilePage;
+
+    test.beforeEach(async ({ page }) => {
+        mainPage = new MainPage(page);
+        loginPage = new LoginPage(page);
+        profilePage = new ProfilePage(page);
+        await mainPage.open();
+    });
+
+    test("Authorization with empty fields", async ({ page }) => {
+        await loginPage.clickEnterButton();
+        await loginPage.clickLoginButton();
+        await page.waitForLoadState("networkidle");
```

Comment:
Using waitForLoadState("networkidle") is a good practice, but consider using specific waits for elements to appear or be hidden when possible to make tests more robust against network delays.